finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/SocketOptionsConfig.scala

```diff
@@ -27,7 +27,7 @@ case class SocketOptionsConfig(
     }
 
     Stack.Params.empty +
-      Transport.Options(noDelay, reuseAddr, reusePort) +
+      Transport.Options(noDelay.getOrElse(true), reuseAddr.getOrElse(true), reusePort.getOrElse(false)) +
```

Comment:
I think it would be good form to put these in some constant such as `DefaultNoDelay`, etc.